src/transformers/modeling_tf_bart.py

```diff
@@ -0,0 +1,1147 @@
+        assert len(encoder_outputs.last_hidden_state.shape) == 3
+        decoder_outputs = self.decoder(
+            decoder_input_ids,
+            encoder_outputs.last_hidden_state,
+            attention_mask,
+            decoder_padding_mask,
+            decoder_causal_mask=causal_mask,
+            decoder_cached_states=past_key_values,
+            use_cache=use_cache,
+            output_attentions=output_attentions,
+            output_hidden_states=output_hidden_states,
+            return_dict=return_dict,
+        )
+        if not return_dict:
+            # Attention and hidden_states will be [] or None if they aren't needed
+            assert isinstance(
+                decoder_outputs[0], tf.Tensor
+            ), f"got type {type(decoder_outputs[0])} for first decoder output"
+            return tuple(x for x in decoder_outputs + encoder_outputs.to_tuple() if x is not None)
+        else:
+            return TFSeq2SeqModelOutput(
+                last_hidden_state=decoder_outputs.last_hidden_state,
+                past_key_values=decoder_outputs.past_key_values,
+                decoder_hidden_states=decoder_outputs.hidden_states,
+                decoder_attentions=decoder_outputs.attentions,
+                encoder_last_hidden_state=encoder_outputs.last_hidden_state,
+                encoder_hidden_states=encoder_outputs.hidden_states,
+                encoder_attentions=encoder_outputs.attentions,
+            )
+
+    def get_input_embeddings(self):
+        return self.shared
+
+    def set_input_embeddings(self, value):
+        self.shared = value
+
+    def get_output_embeddings(self):
+        return self.shared
+
+
+@add_start_docstrings(
+    "The BART Model with a language modeling head. Can be used for summarization.",
+    BART_START_DOCSTRING,
+)
+class TFBartForConditionalGeneration(TFPretrainedBartModel):
+    base_model_prefix = "model"
+    authorized_missing_keys = [
+        r"final_logits_bias",
+        r"encoder\.version",
+        r"decoder\.version",
+        "model.encoder.embed_tokens.weight",
+        "model.decoder.embed_tokens.weight",
+    ]
+
+    def __init__(self, config: BartConfig, *args, **kwargs):
+        super().__init__(config, *args, **kwargs)
+        self.model = TFBartModel(config, name="model")
+        self.use_cache = config.use_cache
+
+    @add_start_docstrings_to_callable(BART_INPUTS_DOCSTRING)
+    @replace_return_docstrings(output_type=TFSeq2SeqLMOutput, config_class=_CONFIG_FOR_DOC)
+    def call(
+        self,
+        inputs,
+        attention_mask=None,
+        decoder_input_ids=None,
+        decoder_attention_mask=None,
+        encoder_outputs: Optional[TFBaseModelOutput] = None,
+        past_key_values=None,
+        labels=None,
+        use_cache=None,
+        output_attentions=None,
+        output_hidden_states=None,
+        return_dict=None,
+        training=False,
+        **kwargs,
+    ):
+        """
+        Returns:
+
+        Examples:
```

Comment:
You need to semicolons: 
```suggestion
        Examples::
```